coalib/bearlib/aspects/collections.py

```diff
@@ -7,7 +7,7 @@ class AspectList(list):
     List-derived container to hold aspects.
     """
 
-    def __init__(self, seq=()):
+    def __init__(self, seq=(), excludes=()):
```

Comment:
Agree for the `exclude=None`. But if we change `seq=None` then it could give an error for declaring empty AspectList.

```python
>>> from coalib.bearlib.aspects.collections import AspectList
>>> AspectList()
Traceback (most recent call last):
    ...
TypeError: 'NoneType' object is not iterable